rancher/kubernetes/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Kubernets in Rancher
+layout: rancher-default
+---
+
+## Kubernetes
+---
+
+To deploy Kubernetes in Rancher, you'll first need to create a new [environment]({{site.baseurl}}/rancher/configuration/environments/) that has specified the cluster management to be Kubernetes. 
+
+### Creating a Kubernetes Environment
+
+In the dropdown of environments, click on the **Manage Environments**. A list of existing environments will be displayed. The _Default_ environment that is started when you launch Rancher is based on the _Cattle_ cluster management. You can choose to change the cluster management type of the _Default_ environment or  create a new environment that uses _Kubernetes_ as the cluster management.
```

Comment:
Have we tested the behavior of migrating a cattle environment with deployed templates to kubernetes container orchestration? This is functionally possible on the UI. It seems like migrating existing containers to kubernetes would be difficult, since even the template configuration schema is different between cattle/k8s.